src/api/goods.ts

```diff
@@ -10,10 +10,12 @@ export function getAll(): Promise<Good[]> {
 
```

Comment:
Let us think about a situation when an error comes from the server. The best way is to throw the error

Reply:
this comment, wasn't fixed